README.md

```diff
@@ -137,6 +148,10 @@ Requirements:
 - The provided DID string MUST be compliant with the `did:itn` method.
 - The provided DID document MUST be compliant with the DID V1.0 DID document specification.
 
+Here is how update operation works:
+
+- The ITN agent is responsible to verify that the initiator of the update request is the controller of the DID subject that will be updated. The authentication method defined in the DID Document will be used to authenticate the controller.
```

Comment:
Update request sender must be specified as a DID Document `controller` and is authenticated by verifying the update request signature. It must be signed by the recovery key which is used before to generate the DID identifier.

Reply:
Also we need to mention that DID Document is updated in CAS and in ledgers.

Reply:
Updated.